galaxy_ng/tests/integration/aap/test_aap_rbac.py

```diff
@@ -0,0 +1,117 @@
+# from galaxykit.utils import wait_for_task
+# from galaxy_ng.tests.integration.utils import set_certification
+
+
+pytestmark = pytest.mark.qa  # noqa: F821
+
+
+@pytest.mark.deployment_standalone
+@pytest.mark.skipif(
+    not os.environ.get('JWT_PROXY'),
+    reason="relies on jwt proxy"
+)
+def test_aap_team_member_vs_admin(
+    settings,
+    ansible_config,
+    galaxy_client,
+    random_username
+):
+    # if settings.get('allow_local_resource_management') is False:
+    #     pytest.skip("this test relies on local resource creation")
+
+    gc = galaxy_client("admin", ignore_cache=True)
+    # ga = BasicAuthClient(gc.galaxy_root, 'admin', 'redhat1234')
+    ga = BasicAuthClient(gc.galaxy_root, 'admin', 'admin')
+
+    org_name = random_username.replace('user_', 'org_')
+    team_name = random_username.replace('user_', 'team_')
+
+    # make the org in the gateway
+    org_data = ga.post(
+        '/api/gateway/v1/organizations/',
+        body=json.dumps({'name': org_name})
+    )
+
+    # make the team in the gateway
+    team_data = ga.post(
+        '/api/gateway/v1/teams/',
+        body=json.dumps({'name': team_name, 'organization': org_data['id']})
+    )
+
+    # make the user in the gateway
+    user_data = ga.post(
+        '/api/gateway/v1/users/',
+        body=json.dumps({'username': random_username, 'password': 'redhat1234'})
+    )
+
+    # get all of gateway's roledefs ...
+    gateway_roledefs = ga.get('/api/gateway/v1/role_definitions/')
+    gateway_roledefs = dict((x['name'], x) for x in gateway_roledefs['results'])
+
+    # get all of galaxy's roledefs ...
+    galaxy_roledefs = ga.get('/api/galaxy/_ui/v2/role_definitions/')
+    galaxy_roledefs = dict((x['name'], x) for x in galaxy_roledefs['results'])
+
+    '''
+    # get galaxy's teams
+    galaxy_teams = ga.get('/api/galaxy/_ui/v2/teams/?page_size=100')
+    galaxy_teams_map = dict((x['name'], x) for x in galaxy_teams['results'])
+    '''
+
+    '''
+    # get galaxy's users
+    galaxy_users = ga.get('/api/galaxy/_ui/v2/users/?page_size=100')
+    galaxy_users_map = dict((x['username'], x) for x in galaxy_users['results'])
+    '''
+
+    # make the user a team member in the gateway ...
+    ga.post(
+        '/api/gateway/v1/role_user_assignments/',
+        body=json.dumps({
+            'user': user_data['id'],
+            'role_definition': gateway_roledefs['Team Member']['id'],
+            'object_id': team_data['id'],
+        })
+    )
+
+    # access galaxy as the user to process claims ...
+    uc = BasicAuthClient(gc.galaxy_root, random_username, 'redhat1234')
+    new_data = uc.get(f'/api/galaxy/_ui/v2/users/?username={random_username}')
+    assert new_data['count'] == 1
+    new_user = new_data['results'][0]
+    assert new_user['username'] == random_username
+    assert not new_user['organizations']
+    new_teams = [x['name'] for x in new_user['teams']]
+    assert new_teams == [team_name]
```

Comment:
So basically, the user was placed in the team by using the gateway, and here, is reflected in /api/galaxy/. That's the key assertion.